web/src/pages/mgmt/modifyBusiness.tsx

```diff
@@ -53,16 +52,16 @@ const ModifyBusinessPage = (): ReactElement => {
               <div>
                 <div className="">The current busiess contains the following license data:</div>
```

Comment:
Pebble: Noticed there's a typo should be `The current business contains the following license data:`